types/node/stream.d.ts

```diff
@@ -18,6 +18,11 @@ declare module "stream" {
         }
 
         class Readable extends Stream implements NodeJS.ReadableStream {
+            /**
+             * A utility method for creating Readable Streams out of iterators.
+             */
+            static from(iterable: Iterable<any>, options?: ReadableOptions): Readable;
```

Comment:
Should the type of iterable be `Iterable<any> | AsyncIterable<any>`?

Reply:
`AsyncIterable` isn't supported by the mimum required node version, I tried adding it as an overload in the ts3.2 folder but that didn't work.

Reply:
Nvm, adding shallow definition.